lib/ohai/plugins/passwd.rb

```diff
@@ -9,7 +10,7 @@
   # @return [String]
   #
   def fix_encoding(str)
-    str.force_encoding(Encoding.default_external) if str.respond_to?(:force_encoding)
+    str.dup.force_encoding(Encoding.default_external) if str.respond_to?(:force_encoding)
```

Comment:
```suggestion
    str.dup.force_encoding(Encoding.default_external)
```

`.force_encoding` was added a long time ago, probably safe to remove.